contracts/NFTs/Base/MultiPrivilege/MultiPrivilegeTransferableBurnable.sol

```diff
@@ -0,0 +1,229 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+import "./IMultiPrivilege.sol";
+import "../NftBaseUpgradeableTransferableBurnable.sol";
+
+/// @title MultiPrivilege
+/// @dev Based on the EIP-5496 https://eips.ethereum.org/EIPS/eip-5496
+abstract contract MultiPrivilege is
+    Initializable,
+    NftBaseUpgradeable,
+    IMultiPrivilege
+{
+    using CountersUpgradeable for CountersUpgradeable.Counter;
```

Comment:
We're repeating a lot here and in `NftBaseUpgradeableTransferableBurnable.sol`. Could we leave an explanation somewhere? I know we've had to do this sort of thing in the past and I can't even remember where. It's getting wild.

Reply:
Like when I extend `ERC721Burnable` I'm getting told right at the point of definition that it's burnable. Here it's not so easy.